app/slackimport.go

```diff
@@ -553,12 +553,63 @@ func SlackConvertChannelMentions(channels []SlackChannel, posts map[string][]Sla
 }
 
 func SlackConvertPostsMarkup(posts map[string][]SlackPost) map[string][]SlackPost {
-	// Convert URLs in Slack's format to Markdown format.
-	regex := regexp.MustCompile(`<([^|<>]+)\|([^|<>]+)>`)
+	regexReplaceAllString := []struct {
+		regex *regexp.Regexp
+		rpl   string
+	}{
+		// URL
+		{
+			regexp.MustCompile(`<([^|<>]+)\|([^|<>]+)>`),
+			"[$2]($1)",
+		},
+		// bold
+		{
+			regexp.MustCompile(`\*([^*]+)\*`),
+			"*$0*",
```

Comment:
This regex should only match where the character before the first * is either the start of a line/message, or whitespace, and the character after the second one is either whitespace or the end of the line/message.

This should also not match when the first and last * characters are on different lines.

#### E.g. 

`*test*` should bold but `stuff*things*stuff` should not contain any bold.

    there is *no bold
    in this*

The above should not bold.

This will then accurately reflect how Slack treats the markup.